cmd/kam/kam.go

```diff
@@ -1,9 +0,0 @@
-package main
```

Comment:
What's the reason for deleting this?

Reply:
Ugh .. I might have deleted that by mistake. 
Added the file !